package.json

```diff
@@ -1,21 +1,63 @@
 {
   "name": "@stoplight/types",
-  "version": "1.2.2",
-  "author": "Stoplight <support@stoplight.io>",
-  "license": "MIT",
+  "version": "0.0.0",
+  "description": "Common typings for the Stoplight ecosystem.",
+  "keywords": [],
   "main": "./index.d.ts",
   "types": "./index.d.ts",
-  "scripts": {
-    "build": "tsc --project tsconfig.json"
+  "sideEffects": false,
+  "homepage": "https://github.com/stoplightio/types",
+  "bugs": "https://github.com/stoplightio/types/issues",
+  "author": "Stoplight <support@stoplight.io>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stoplightio/types.git"
   },
-  "prettier": {
-    "printWidth": 100,
-    "singleQuote": true,
-    "trailingComma": "es5"
+  "license": "Apache-2.0",
+  "files": [
+    "**/*",
+    "!__tests__"
+  ],
+  "engines": {
+    "node": ">=8.3.0"
+  },
+  "scripts": {
+    "build.docs": "typedoc --out docs-auto --target es6 --theme minimal --mode modules --excludeNotExported --includeDeclarations",
+    "commit": "git-cz",
+    "lint": "sl-scripts lint",
+    "lint.fix": "yarn lint --fix",
+    "release": "sl-scripts release",
+    "release.docs": "sl-scripts release:docs",
+    "release.dryRun": "sl-scripts release --dry-run --debug"
   },
   "devDependencies": {
-    "tslint": "5.11.x",
-    "tslint-config-stoplight": "1.1.x",
-    "typescript": "3.1.x"
+    "@stoplight/scripts": "1.1.3",
+    "typescript": "3.1.6"
+  },
+  "lint-staged": {
+    "*.{ts,tsx}$": [
+      "yarn lint.fix",
+      "git add"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+      "pre-commit": "lint-staged"
+    }
+  },
+  "config": {
+    "commitizen": {
+      "path": "node_modules/cz-conventional-changelog"
+    }
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
```

Comment:
Isn't `husky`, `config` and `commitlint` already defined in `stoplight/scripts`?

Reply:
Yeah but I'm not sure they'll get picked up since those definitions are in the package.json file in node_modules? That would be awesome if they were, then could remove all this boilerplate that is on all of our repos.

Reply:
I do not think they're picked up automatically. We can try it but as far as I remember they're not.